ITKImageProcessingFilters/itkImportImageMallocContainer.h

```diff
@@ -0,0 +1,203 @@
+/*=========================================================================
+*
+*  Copyright Insight Software Consortium
+*
+*  Licensed under the Apache License, Version 2.0 (the "License");
+*  you may not use this file except in compliance with the License.
+*  You may obtain a copy of the License at
+*
+*         http://www.apache.org/licenses/LICENSE-2.0.txt
+*
+*  Unless required by applicable law or agreed to in writing, software
+*  distributed under the License is distributed on an "AS IS" BASIS,
+*  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+*  See the License for the specific language governing permissions and
+*  limitations under the License.
+*
+*=========================================================================*/
+#ifndef itkImportImageMallocContainer_h
+#define itkImportImageMallocContainer_h
+
+#include "itkObject.h"
+#include "itkObjectFactory.h"
+#include <utility>
+
+namespace itk
+{
+  /** \class ImportImageMallocContainer
+  *  \brief Defines an itk::Image front-end to a standard C-array.
+  *
+  * Defines an itk::Image front-end to a standard C-array. This container
+  * conforms to the ImageContainerInterface. This is a full-fleged Object,
+  * so there is modification time, debug, and reference count information.
+  *
+  * \tparam TElementIdentifier An INTEGRAL type for use in indexing the
+  * imported buffer.
+  *
+  * \tparam TElement The element type stored in the container.
+  *
+  * \ingroup ImageObjects
+  * \ingroup IOFilters
+  * \ingroup ITKCommon
+  */
+
+  template< typename TElementIdentifier, typename TElement >
+  class ImportImageMallocContainer :public Object
```

Comment:
:+1:  After looking into it again, I don't think there is a reason to no derive from ImportImageContainer.